Automattic/jetpack
# Update/lower sync hook priority

Related to https://github.com/Automattic/jpop-issues/issues/8136 which was closed. 

## Proposed changes:

Based on [ticket](https://github.com/Automattic/jpop-issues/issues/8136) it seemed that plugins might be doing a select, using the result and not freeing it properly before we run the Sync code. The idea is to lower the priority to **9998** ( Similarly to  `full_sync` which is assigned **9999**) to try and ensure that sync gets executed at the end.

### Other information:


## Jetpack product discussion


p1678439449940289-slack-CBG1CP4EN

## Does this pull request change what data or activity we track or use?



## Testing instructions:






1. First of all, let's make sure we are not breaking sync.
* Do some changes to the site and check in ES that sync is being run as usual.
* For example add a new post and check that you get a record with the feature `async_publish_post`.

2. Test by adding another action with lower priority and make sure it is executed prior to `sync` and `full_sync` methods.
